MapboxNavigation/NavigationViewController.swift

```diff
@@ -200,6 +200,26 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
     
     var bottomViewController: ContainerViewController?
     
+    public typealias FloatingButtonsPositions = NavigationView.FloatingButtonsPositions
+    
+    open var floatingButtonsPosition: FloatingButtonsPositions? {
```

Comment:
Remember to add documentation about these two public properties and also mention them in the changelog.